front/poke/temporal/client.ts

```diff
@@ -0,0 +1,23 @@
+import { getTemporalClient } from "@app/lib/temporal";
+
+import { scrubDataSourceWorkflow } from "./workflows";
```

Comment:
NIT: relative import?

Reply:
For some reason this one _has_ to be a relative import. I don't fully understand it, but it has to do with the way temporal bundles the workflow code (with webpack I think)

Reply:
I added this comment and then I saw "workflowsPath: require.resolve("./workflows")"  in front/poke/temporal/worker.ts and I wondered if that was the reason indeed. Thanks for the answer 🙇🏻‍♀️